src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -169,6 +179,8 @@ function getDrupalContent(buildOptions) {
       buildOptions.drupalData = drupalData;
       done();
     } catch (err) {
+      if (err instanceof ReferenceError) throw err;
```

Comment:
This is so the build breaks when we reference a feature flag that doesn't exist.